Source/WebCore/Modules/indexeddb/server/UniqueIDBDatabaseTransaction.cpp

```diff
@@ -77,21 +81,31 @@ void UniqueIDBDatabaseTransaction::abort()
 {
     LOG(IndexedDB, "UniqueIDBDatabaseTransaction::abort");
     
-    auto database = databaseConnection().database();
-    ASSERT(database);
+    if (!database())
+        return;
 
-    database->abortTransaction(*this, [this](auto& error) {
+    database()->abortTransaction(*this, [this, weakThis = WeakPtr { *this }](auto& error) {
```

Comment:
We are calling twice database() here.
How about:
auto database = this->database();
if (!database)
    return;
database->...

Reply:
Sounds good; will change.